frontend/src/components/TaskMapEdge.tsx

```diff
@@ -25,8 +66,9 @@ const DrawPath: FC<any> = ({ id, d, markerEnd }) => {
         className={`react-flow__edge-path ${classes(css.invisiblePath)}`}
         d={d}
         markerEnd={markerEnd}
-        // eslint-disable-next-line no-alert
-        onClick={() => alert(`To do: remove connections = ${id}`)}
+        onContextMenu={handleDelete}
```

Comment:
```suggestion
        onAuxClick={handleDelete}
        onContextMenu={(e) => { e.preventDefault(); }}
```

The context menu seems to trigger after mouse down, not after a click (down + up), using the `onAuxClick` event might be nicer. Then just check if the mouse button is the right one (pun intended)